src/translate.rs

```diff
@@ -1003,6 +1003,88 @@ where
     }
 }
 
+pub struct PtrArray(*mut glib_sys::GPtrArray);
+
+impl Drop for PtrArray {
+    fn drop(&mut self) {
+        unsafe {
+            glib_sys::g_ptr_array_free(self.0, false.to_glib());
```

Comment:
This should be with `true`, otherwise you leak memory. But more important, `gst_ptr_array_unref()`

Reply:
You are indeed correct. I looked into it more closely and found a case where an array was passed as reference and it was leaking. It is changed now to `true` and does not leak anymore.